pipeline-braille-scripts/xml-to-pef/src/main/README.md

```diff
@@ -0,0 +1,12 @@
+# xml-to-pef API
+
+## <a href="resources/META-INF/catalog.xml" class="source">catalog.xml</a>
+
+## OSGi services
+
+### Scripts (`org.daisy.pipeline.script.XProcScriptService`)
+
+- [`http://www.daisy.org/pipeline/modules/braille/xml-to-pef/xml-to-pef.xpl`](resources/xml/xproc/xml-to-pef.xpl)
```

Comment:
Ok, so I can delete this file? I wasn't quite sure what these files did but it seemed like something you use to generate documentation so I just copy/pasted and replaced format names and hoped for the best.